### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by 123liluky** [sujoyp/wtalc-pytorch#8]
I used my own small dataset to extract I3D features. Then use main.py to train. Overfitting happened in training set. Do you have any suggestions?
![image](https://user-images.githubusercontent.com/20832778/62195016-a75b3f80-b3ad-11e9-9c84-71592b2f2bd4.png)

**Comment by sujoyp** [comment]
It will hard to comment without more specifications about the dataset, train and test set size, etc.
